### PR TITLE
Enable Short URLs starting with 1.32

### PR DESCRIPTION
**Pull request by davidbarratt**

### CODE REVIEW

File: 1.32/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+# Install the PHP extensions we need
+RUN docker-php-ext-install mbstring mysqli opcache intl
+
+# Install the default object cache.
+RUN pecl channel-update pecl.php.net \
+	&& pecl install apcu \
+	&& docker-php-ext-enable apcu
+
+# Enable Short URLs
```
Comment (addshore) [inline 1.32/Dockerfile]: I just tested this in my own image and it seems to work pretty well.

Comment (davidbarratt) [reply 1.32/Dockerfile]: Thanks for testing!

**Comment by davidbarratt** [comment]
Merging this so we don't forget about it when the new release is available. :)

**Comment by magol** [comment]
https://phabricator.wikimedia.org/T204244

**Comment by shunkica** [comment]
Sorry for posting here but I do not understand how to permanently enable short url in the docker image and can't find any info about it other than this pull request.

**Pull request merged**